### ISSUE TITLE
Copyright????

### ISSUE THREAD

**Issue by Nozlaf** [denetwerkspecialist/Auctions#5]
how can you "copyright" code from another open source project as your own?

**Comment by MESWEB** [comment]
Yeah that's good question. He just copy all files and he change header to his own where is that p..e of s...t:
```php
This code is proparty of The Networkspecialist.com.
 *   Auctions Pro users must have a valid license to use this program. 
```
LOL

**Comment by denetwerkspecialist** [comment]
It is simple Webid = DEAD - Had many Errors/Bugs
It is still Open Source , but we want so "see"how many people are using it so that why we added the license system, the github version is free, everybody can get as many license as they want.
We fixed many bugs
We rewrote many parts of the code and added many new things improved the install script
Yes we do got a Paid version where we added new stuff with our from us
So what is the problem here?

**Comment by Nozlaf** [comment]
you have no right to copyright my work, it is not "proparty of you" you have no right to claim the project is dead it is a open source project which no-one is actively maintaining currently,
you cannot remove our copyright notice without permission, please return our copyright notice to your project

**Comment by Nozlaf** [comment]
@denetwerkspecialist 

I've submitted DMCA takedown against your project as it clealy infringes on our copyright, should you wish to avoid having to takedown your project you have the option to return attribution of our work to our project on all the files.

**Comment by MESWEB** [comment]
> We rewrote many parts of the code and added many new things improved the install script
Yes we do got a Paid version where we added new stuff with our from us
Open Source users can upgrade to the Pro version with a license upgrade so we have one set of code instead of two version
So what is the problem here?

@denetwerkspecialist: In your repo many files are not touched - same code 1:1 and just changed heading. It's same like copy Windows and change name and selling as new product. You should ask for permission. Many people spend long time to make this great code and now you try sell their work as your own project.

**Comment by denetwerkspecialist** [comment]
@Nozlaf @MESWEB  we will delete the code and wish you good luck this buggy software
